src/main/java/org/example/Chair.java

```diff
@@ -1,5 +1,14 @@
 package org.example;
 
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
These imports are unused and should be removed to reduce clutter.

Reply:
Yeah I forgot to remove these!
Thanks!

Reply:
Are you sure?
I used them on both methods!

Reply:
Correct, but the methods you are talking about are in the Extractor class. This is the Chair class which is a simple data structure that needs none of these imports.

Reply:
Got it!